i18n/i18n.go

```diff
@@ -84,7 +86,12 @@ func (i18n *I18n) T(locale, key string, args ...interface{}) string {
 		value = translations[translationKey].Value
 	} else {
 		// Save translations
-		i18n.SaveTransaltion(&Translation{Key: translationKey, Locale: locale, Backend: i18n.Backends[0]})
+		err := i18n.SaveTranslation(&Translation{Key: translationKey, Locale: locale, Backend: i18n.Backends[0]})
+		log.Printf("Error saving translation: [%s]: %s\n", locale, translationKey)
+		if err != nil {
+			delete(translations, translationKey)
```

Comment:
@jinzhu if i dont delete the key here it is cached somewhere and actually appears in the frontend or the translation admin interface - although it's not stored in the db. this works  - but ideally it would not go in if there was a save error... couldnt find out where that happens.